crates/client/rpc/src/lib.rs

```diff
@@ -2,6 +2,7 @@
 //!
 //! It uses the madara client and backend in order to answer queries.
 
+pub mod astarknet;
```

Comment:
Can we think of a better name than `astrarknet` I'm not sure of what you meant by that.
Can we name it `StarknetRpcWrapper` or something along those lines?

Reply:
I called it Astarknet for short of arcStarknet but I can just change it to StarknetRpcWrapper. Do you want to change the name of the file, the mod and every instance of the name?

Reply:
`StarknetRpcWrapper` is not a really good name either haha